packages/drivers/local-driver/src/localDocumentDeltaConnection.ts

```diff
@@ -79,9 +79,13 @@ export class LocalDocumentDeltaConnection extends DocumentDeltaConnection {
 
 	/**
 	 * Submits a new signal to the server
+	 *
+	 * @param content - Content of the signal.
+	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	public submitSignal(message: any): void {
-		this.emitMessages("submitSignal", [[message]]);
+	public submitSignal(content: any, targetClientId?: string): void {
```

Comment:
Add keyword `override` to help make it clear this is overriding content in super.
(Do you even need this to make the test change? Perhaps don't change anything but test code unless you need to.)

Reply:
Yeah it isn't necessary, will remove these changes.